libs/core/langchain_core/runnables/graph.py

```diff
@@ -8,6 +8,7 @@
     Any,
     Callable,
     Dict,
+    Hashable,
```

Comment:
Is there a better type? We just need something that defines __str__ ?

Reply:
yup, updated!